tests/rpc/rpc_test.go

```diff
@@ -135,6 +135,33 @@ func callWithError(method string, params interface{}) (*Response, error) {
 	return rpcRes, nil
 }
 
+func TestWeb3_Sha3(t *testing.T) {
```

Comment:
can you move the tests to this file and use the client RPC instead?
https://github.com/tharsis/ethermint/blob/main/tests/e2e/integration_test.go

Reply:
@fedekunze sure, updated